Cannot find module 'estraverse-fb'

While trying to `git commit` a fix for https://github.com/seegno/clean-deep/issues/15, I came across this bug:

```
Error: Cannot find module 'estraverse-fb'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at monkeypatch (/Users/zeke/forks/clean-deep/node_modules/babel-eslint/index.js:59:32)
    at Object.exports.parse (/Users/zeke/forks/clean-deep/node_modules/babel-eslint/index.js:422:5)
    at parse (/Users/zeke/forks/clean-deep/node_modules/eslint/lib/eslint.js:661:31)
    at EventEmitter.module.exports.api.verify (/Users/zeke/forks/clean-deep/node_modules/eslint/lib/eslint.js:799:27)
    at processText (/Users/zeke/forks/clean-deep/node_modules/eslint/lib/cli-engine.js:264:31)
    at processFile (/Users/zeke/forks/clean-deep/node_modules/eslint/lib/cli-engine.js:303:18)
    at executeOnFile (/Users/zeke/forks/clean-deep/node_modules/eslint/lib/cli-engine.js:674:25)
pre-commit: 
pre-commit: We've failed to pass the specified git pre-commit hooks as the `lint`
pre-commit: hook returned an exit code (1). If you're feeling adventurous you can
pre-commit: skip the git pre-commit hooks by adding the following flags to your commit:
pre-commit: 
pre-commit:   git commit -n (or --no-verify)
pre-commit: 
pre-commit: This is ill-advised since the commit is broken.
pre-commit: 
```

This did not do the trick:

```
rm -rf node_modules && npm i
```

I worked around it by running:

```
npm install estraverse-fb
```

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
   "devDependencies": {
     "babel-cli": "^6.16.0",
     "babel-core": "^6.16.0",
-    "babel-eslint": "^4.1.3",
+    "babel-eslint": "^6.1.2",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-preset-es2015-node4": "^2.1.0",
     "eslint": "^3.6.1",

